cmd/api/src/test/integration/database.go

```diff
@@ -59,6 +59,19 @@ func SetupDB(t *testing.T) database.Database {
 	return dbInst
 }
 
+// SetupTransaction returns a *gorm.DB which can be useful when writing integration tests that require a transaction
+func SetupTransaction(t *testing.T) *gorm.DB {
```

Comment:
Is this being used still or is it from a previous iteration and we can clean it up?